src/app/components/list/list.service.js

```diff
@@ -41,16 +37,24 @@
 
     // Cache a list for future offline use
     List.prototype.cache = function () {
+      var lfusers = $localForage.instance('users');
+      var lflists = $localForage.instance('lists');
       var request = { limit: 50, offset: 0, sort: 'name'};
+      lflists.setItem(this._id, this);
```

Comment:
If we caught the error and sent it to new relic we could see if it was happening often and re-assess how much we try to store (but I may be over thinking this)